operations/jsonnet/microservices/ingester.libsonnet

```diff
@@ -41,13 +42,13 @@
     ]) +
     $.util.withResources($._config.ingester.resources) +
     $.util.readinessProbe +
-    (if $._config.variables_expansion then container.withArgsMixin(['--config.expand-env=true']) else {}),
+    (if $._config.variables_expansion then container.withArgsMixin(['-config.expand-env=true']) else {}),
 
-  tempo_ingester_statefulset:
+  newIngesterStatefulSet(name, container, with_anti_affinity=true)::
     statefulset.new(
-      target_name,
-      $._config.ingester.replicas,
-      self.tempo_ingester_container,
+      name,
+      3,
```

Comment:
restore the config option?

Reply:
number of replicas are used in line 72.
https://github.com/grafana/tempo/pull/1936/files#diff-5e48e5df377596900089ccc95e01af67be6cdf749467f4cc0429fa5885ed633eR72